{"body": "@rubys I think I was the one who got it wrong. Load fixture != put them in the database.\r\n\r\nSo it seems like we didn't even create records in the database when we ran `rake test` though. Let me fix it.\r\n\r\nThanks @dhh @rubys and @tenderlove, and I'm sorry that I got it wrong.", "url": "https://api.github.com/repos/rails/rails/issues/comments/15249693", "created_at": "2013-03-21T16:34:30Z", "html_url": "https://github.com/rails/rails/pull/9080#issuecomment-15249693", "updated_at": "2013-03-21T16:34:30Z", "repo": "rails", "issue_id": 9080, "user": {"following_url": "https://api.github.com/users/sikachu/following{/other_user}", "events_url": "https://api.github.com/users/sikachu/events{/privacy}", "organizations_url": "https://api.github.com/users/sikachu/orgs", "url": "https://api.github.com/users/sikachu", "gists_url": "https://api.github.com/users/sikachu/gists{/gist_id}", "html_url": "https://github.com/sikachu", "subscriptions_url": "https://api.github.com/users/sikachu/subscriptions", "avatar_url": "https://2.gravatar.com/avatar/f1c4a3bb1606cc4a61711e61e2fe6146?d=https%3A%2F%2Fidenticons.github.com%2F4a4526b1ec301744aba9526d78fcb2a6.png", "repos_url": "https://api.github.com/users/sikachu/repos", "received_events_url": "https://api.github.com/users/sikachu/received_events", "gravatar_id": "f1c4a3bb1606cc4a61711e61e2fe6146", "starred_url": "https://api.github.com/users/sikachu/starred{/owner}{/repo}", "login": "sikachu", "type": "User", "id": 4912, "followers_url": "https://api.github.com/users/sikachu/followers"}, "owner": "rails", "_id": {"$oid": "523d5a22bd354364b900609b"}, "id": 15249693, "issue_url": "https://api.github.com/repos/rails/rails/issues/9080"}
